configs/openmrs/initializer_config/attributetypes/attribute_types.csv

```diff
@@ -0,0 +1,3 @@
+Uuid,Void/Retire,Entity name,Name,Description,Min occurs,Max occurs,Datatype classname,Datatype config,Preferred handler classname,Handler config,_order:1000
+a56959b5-4fe2-4d69-a553-fdc77957c48b,,Visit,Mode of arrival,The mode of arrival that the patient took to the location,0,1,org.openmrs.customdatatype.datatype.ConceptDatatype,f61a8a18-cb64-4a63-bc28-56b4789941ff,,,
+d63554a8-4979-4719-a918-1ccc3d924ed7,,Visit,Level of emergency severity assessment,Level of emergency severity assessment,0,1,org.openmrs.customdatatype.datatype.ConceptDatatype,bbb02557-0277-4979-a1fa-b0020f136066,,,
```

Comment:
Hi @kazlaw!
The `Level of emergency severity assessment` concept is being used in the `triage.json` form, and I have used the same UUID for the above visit attribute type's `Datatype config` mapping. But I didn't find the concept defined anywhere, do we need to define the concept mentioned above?
Thanks!